### PR TITLE
Pacific Trails Case Study #2

### PR DESCRIPTION
**Pull request by brianandreasen1983**
Submission of part 2 of the case study for Pacific Trails.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Review for Mary.

**Comment by brianandreasen1983** [comment]
Final submission after reviewed corrections

File: yurts.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+    <div id="yurthero">
+    </div>
+    <main>
+
+        <h2>The Yurts at the Pacific Trails</h2>
+
+        <dl>
+            <dt><strong>What is a Yurt?</strong></dt>
```
Comment (mbMosman) [inline yurts.html]: These should be converted to headings and paragraphs to match the activities page.

File: yurts.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <li><a href="Reservations.html">Reservations</a></li>
+        </ul>
+    </nav>
+
+
+    <div id="yurthero">
+    </div>
+    <main>
+
+        <h2>The Yurts at the Pacific Trails</h2>
```
Comment (mbMosman) [inline yurts.html]: Still have the extra "the" in the heading.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        <dl>
+            <dt><strong>What is a Yurt?</strong></dt>
+
+            <dd>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</dd>
+
+            <dt><strong>How are the yurts furnished?</strong></dt>
+
+            <dd>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove.The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</dd>
```
Comment (mbMosman) [inline yurts.html]: Space missing after the .

File: activities.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <div id="trailhero">
+    </div>
+
+    <main>
+
+        <h2> Activities at Pacific Trails</h2>
+
+        <h3>Hiking</h3>
+
+        <p>Pacific Trails Resort has 5 miles of hiking trails and is adjacent to a skate park. Go it alone or join one of our guides hikes.</p>
```
Comment (mbMosman) [inline activities.html]: It's adjacent to a "state" park, not a skate park.

File: pacific.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    padding-left: 30px;
+    padding-bottom: 20px;
+    display: block;
+}
+
+main ul {
+    list-style-image: url(images/marker.gif);
+}
+
+dt {
```
Comment (mbMosman) [inline pacific.css]: The styles for dt can be removed with the dl/dt/dd from the yurts page, but you also need this color for the h3's so you could just change this to h3 if you update the yurts page.

**Comment by mbMosman** [comment]
Make sure to follow through on the corrections or you'll get dinged for the same things each time.

**Pull request closed without merging**